src/components/PostForm/FieldRenderer/FieldRenderer.stories.tsx

```diff
@@ -186,8 +235,7 @@ Filled.args = {
 
 export const Required = Template.bind({})
 Required.args = {
-  minimum: 1,
-  maximum: 1
```

Comment:
was there a reason why this line was removed?

Reply:
Current required args is strictly 1, which doesn't make sense for a multiple select component